src/content/blog/themes-catalog-updates.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Themes Catalog Updates'
+description: 'Announcing the new and improved themes catalog. Now with support for more detailed theme descriptions and premium themes.'
+publishDate: 'December 6, 2022'
+authors:
+    - tony
+socialImage: '/assets/blog/themes-catalog-updates/blog-social.jpg'
+coverImage: '/assets/blog/themes-catalog-updates/blog-hero.jpg'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro'
+import CallToAction from '/src/components/CallToAction.astro'
+import Note from '/src/components/Note.astro'
+import themesPreview from '/src/assets/blog/themes-catalog-updates/themes-preview.png'
+import themesPreview2 from '/src/assets/blog/themes-catalog-updates/themes-preview.png'
+import paidThemes from '/src/assets/blog/themes-catalog-updates/paid-themes.png'
+import submitThemes from '/src/assets/blog/themes-catalog-updates/submit-themes.png'
+
+We [launched Astro Themes](https://astro.build/blog/themes-and-integrations/) earlier this year to highlight the growing list of Astro themes and starter templates being developed by our amazing community. Astro's [Themes Catalog](https://astro.build/themes/) makes it easy to browse a collection of themes for your next site and get started with a full-featured project from day one.
+
+**Today, we're excited to announce improvements to the themes catalog, support for paid themes, and an improved [theme submission process](https://astro.build/themes/submit).**
+
+<BlogContentImage
+    src={themesPreview}
+    alt="Screenshots of three Astro theme home pages"
+/>
+
+## Theme Detail Pages
+
+The initial release of our themes catalog focused on creating a launchpad for Astro themes, giving a quick overview of themes with a preview image and short description. As the list of themes grew that lead to an obvious question, what if you want to know more about a specific theme?
+
+<BlogContentImage
+    src={themesPreview}
+    alt="Screenshots of the new Astro themes catalog details page"
+/>
```

Comment:
Looks like this is the same image as immediately above. Should they be different?

<img width="410" alt="image" src="https://user-images.githubusercontent.com/357379/205976899-430b0665-0baa-4733-aa9b-937abc861321.png">

Reply:
Yep, that should have been a separate image.  Thanks!